editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -85,8 +85,11 @@ void AnimationPlayerEditor::_notification(int p_what) {
 				track_editor->set_anim_pos(player->get_current_animation_position());
 				EditorNode::get_singleton()->get_inspector()->refresh();
 
+			} else if (!player->is_valid()) {
+				// Reset timeline when the player has been stopped externally
+				frame->set_value(0);
 			} else if (last_active) {
-				// Need the last frame after it stopped.
+				// Need the last frame after it stopped
```

Comment:
It's actually the current consensus that we do end comments with a dot.

Reply:
Sorry for that, just reverted it.

Reply:
On that topic, could you add a dot to the end of the comment you added?

Reply:
Yep. Good point.

Reply:
Also, I literally didn't pay attention that this was already merged, sorry... :P